spec/reek/smells/nested_iterators_spec.rb

```diff
@@ -43,7 +75,7 @@ def chained
         @sig.keys.sort_by { |xray| xray.to_s }.each { |min| md5 << min.to_s }
       end
     EOS
-    expect(src).not_to reek_of(:NestedIterators)
+    expect(src).not_to reek_of(described_class)
```

Comment:
Hmm, I kind of agree but given that we use this consistently everywhere it still should be obvious, don't you think?
Overall I'm not hung up on that and I can switch it back if you feel that this decreases readability signficantly.